src/Plugin/Psalm.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace PHPCensor\Plugin;
+
+use PHPCensor;
+use PHPCensor\Builder;
+use PHPCensor\Model\Build;
+use PHPCensor\Model\BuildError;
+use PHPCensor\Plugin;
+use PHPCensor\ZeroConfigPluginInterface;
+
+/**
+ * A static analysis tool for finding errors in PHP applications https://getpsalm.org
+ *
+ * Class Psalm
+ * @package PHPCensor\Plugin
+ */
+class Psalm extends Plugin implements ZeroConfigPluginInterface
```

Comment:
You shouldn't use `ZeroConfigPluginInterface` and `zero_config` option for this plugin. It will break builds without configuration on PHP5.6.

Reply:
@corpsee Can you please explain a bit more why will break configuration for PHP 5.6?

Reply:
`canExecuteOnStage` could always return `false` if PHP < 7

Reply:
@garas & @corpsee tbh, I did not read the code of PHP Censor, I just focused on the plugin and just took a few things from other plugins as a reference. I can't yet understand the role of `ZeroConfigPluginInterface` and the `canExecuteOnStage` and why it will return `false` and why that will break compatibility, but since you both say so, then what I should use instead of `ZeroConfigPluginInterface` or what should I do to maintain the compatibility for PHP 5.6 and keep `ZeroConfigPluginInterface` if that is necessary?